### DISCUSSION TITLE
gulpfile.js

### DISCUSSION THREAD

**Discussion by abdotech1** [holger1411/understrap#1688] [Q&A]
hello guys! please i didn't find gulpfile.js in the software any help with that?

**Comment by bacoords** [comment]
Hi @abdotech1 - Understrap moved away from gulp for it's build process. Please review the [documentation](https://docs.understrap.com/#/understrap-child/npm) or check the package.json file for more info on how to run the build scripts.

**Comment by abdotech1** [comment]
hi @bacoords - yeah i do what the doc say but when i come to change in sass files nothing happened in front

**Reply by bacoords** [reply]
Can you show us the results of your console?

**Reply by bacoords** [reply]
I'd also recommend including the results of `node -v`
